governance/api.go

```diff
@@ -114,13 +109,12 @@ func (api *PublicGovernanceAPI) Vote(key string, val interface{}) (string, error
 }
 
 func (api *PublicGovernanceAPI) isRemovingSelf(val interface{}) bool {
-	target := val.(string)
-
-	if common.HexToAddress(target) == api.governance.nodeAddress.Load().(common.Address) {
-		return true
-	} else {
-		return false
+	for _, str := range strings.Split(val.(string), ",") {
```

Comment:
1. Can we ensure that `val` cannot be `nil`? If it is, this line will cause a panic.
2. I think we need to trim `str`. If not, `common.HexToAddress(str)` can return invalid values.

Reply:
> Can we ensure that val cannot be nil? If it is, this line will cause a panic.

`val` cannot be `nil` because those votes are filtered by `api.governance.ValidateVote(&GovernanceVote{Key: key, Value: val})`.

> I think we need to trim str. If not, common.HexToAddress(str) can return invalid values.

Currently,, the vote api does not accept the `str` which contains whitespaces.. thanks for the advice. Reflected at https://github.com/klaytn/klaytn/pull/1110/commits/abdc7e4170e95f7550593e1e957cd0f762258529.